lib/wakes/location_string.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module LocationString
```

Comment:
@benhutton Won't `URIFromLocationString.new` return an instance of URIFromLocationString, rather than a URI? 

I think it would need to be
```
class URIFromLocationString
  def self.generate(location_string)
    location_string = "http://#{location_string}" if URI(location_string).scheme.nil?
    URI(location_string)
  end
end
```

and then `URIFromLocationString.generate('/some/path').host`